app/controllers/users/piv_cac_authentication_setup_controller.rb

```diff
@@ -104,11 +104,7 @@ def process_valid_submission
       Funnel::Registration::AddMfa.call(current_user.id, 'piv_cac')
       session[:needs_to_setup_piv_cac_after_sign_in] = false
       final_path = after_sign_in_path_for(current_user)
-      next_mfa_setup_for_user = user_session.dig(
-        :selected_mfa_options,
-        determine_next_mfa_selection,
-      )
-      redirect_to user_next_authentication_setup_path(next_mfa_setup_for_user) ||
+      redirect_to next_setup_path ||
                   final_path
```

Comment:
more style (general rule of thumb, I wouldn't add a line break to something unless it was hitting the 100c limit)
```suggestion
      redirect_to next_setup_path || final_path
```